### ISSUE TITLE
Adopt CIOU/DIOU as a module

### ISSUE THREAD

**Issue by NoOneUST** [Zzh-tju/DIoU-SSD-pytorch#18]
Hello,

I am writing a instance segmentation network and I adopt a SSD head as the detection head. If I want to adopt CIOU/DIOU, how should I modify my code to integrate them together? Can this repo be directly called as a module? Thank you!

**Comment by Zzh-tju** [comment]
I dont know how to integrate the models together. If you want to use D/CIoU loss, you just need to modify the box output to caculate the loss.

**Comment by NoOneUST** [comment]
Can I just use class MultiBoxLoss(nn.Module) in this file to replace my original detection loss and everything is done?
https://github.com/Zzh-tju/DIoU-SSD-pytorch/blob/master/utils/loss/multibox_loss.py

**Comment by Zzh-tju** [comment]
feel free to try anything until you can get a stable training. my advice may not be helpful
